src/library/Uno.Material/Styles/Controls/v2/NavigationView.xaml

```diff
@@ -1205,7 +1205,17 @@
 							</Grid.RowDefinitions>
 
 							<!-- DisplayMode top -->
-							<StackPanel x:Name="TopNavArea" />
+							<StackPanel x:Name="TopNavArea">
+
+								<!-- As of Microsoft.UI.Xaml 2.6.0-prerelease.210430001, PaneTitleOnTopPane must be present in the template -->
+								<ContentControl x:Name="PaneTitleOnTopPane"
```

Comment:
dont bother with properties on this, just keep the bare minimum xName+Visibility